tests/rptest/clients/rpk.py

```diff
@@ -360,7 +360,15 @@ def parse_partition(string):
 
         def try_describe_group(group):
             cmd = ["describe", group]
-            out = self._run_group(cmd)
+            try:
+                out = self._run_group(cmd)
+            except RpkException as e:
+                if "COORDINATOR_NOT_AVAILABLE" in e.msg:
+                    # Transient, return None to retry
+                    return None
```

Comment:
Can some logging be useful here to record what's going on?

Reply:
I think we already have the raw RPK output in debug logs -- in the event of a failure we can see what happened.